### PR TITLE
Fz83 p 9

### PR DESCRIPTION
**Pull request by Decomplosed**
Pls check the team info section

### CODE REVIEW

File: assets/style/style.css [hunk 1]
```diff
@@ -356,10 +356,86 @@ h2 {
     .add-info__container {
         flex-direction: row;
         display: flex;
+        flex-direction: row;
```
Comment (bartzaw) [inline assets/style/style.css:359]: double flex-direction in this class? probably some left-over after merge

File: assets/style/style.css [hunk 2]
```diff
@@ -356,10 +356,86 @@ h2 {
...
+    font-size: 1em;
+}
+
+.icon-social{
+    padding-top: 10px;
+    border-top: 1px solid #000000;
+}
+
+.team-member-container{
+    opacity: 0;
```
Comment (bartzaw) [inline assets/style/style.css:435]: why there's 0 opacity for container? Was it intended?

File: assets/style/style.css [hunk 3]
```diff
@@ -356,10 +356,86 @@ h2 {
...
+}
+
+.team-member:hover .team-member-container{
+    opacity: 1;
```
Comment (bartzaw) [inline assets/style/style.css:439]: and here's opacity: 1?

**Review by bartzaw — CHANGES_REQUESTED** [review]
Please see my comments to the code.

File: assets/style/style.css [hunk 4]
```diff
@@ -366,7 +367,8 @@ h2 {
 
 .team-info{
     display: flex;
-    flex-grow: 1;
+    width: 80vw;
```
Comment (icelandico) [inline assets/style/style.css]: `vw` is a tricky unit. Shall look good but maybe `%` are better choice?

File: assets/style/style.css [hunk 5]
```diff
@@ -363,3 +363,12 @@ h2 {
     }
 
 }
+
+
+.team-text-container{
+    text-align: center;
+    margin-top: 1em;
+    margin-bottom: 2em;
```
Comment (icelandico) [inline assets/style/style.css:377]: Suggestion: put margin styling in one line.

**Review by icelandico — APPROVED** [review]
Minor changes required.

**Comment by Decomplosed** [comment]
Closed this pull request - moved changes from team section to another branch

**Pull request closed without merging**